envbuilder.go

```diff
@@ -1204,13 +1001,14 @@ func (fs *osfsWithChmod) Chmod(name string, mode os.FileMode) error {
 	return os.Chmod(name, mode)
 }
 
-func findDevcontainerJSON(options Options) (string, string, error) {
+func findDevcontainerJSON(options Options, fs billy.Filesystem, logf Logger) (string, string, error) {
```

Comment:
```suggestion
func findDevcontainerJSON(fs billy.Filesystem, logf Logger, options Options) (string, string, error) {
```
(See other comment, but ideally we'd move these back into options IMO.)

Reply:
Going to add the logf and fs back into the Options